src/apps/persistence/workspace.clj

```diff
@@ -24,16 +26,34 @@
        (map-indexed (partial app-groups/add-subgroup root-category-id))
        (dorun)))
 
+(defn set-workspace-root-app-group
```

Comment:
Thanks. No, the linter is configured not to look for dead code because it was reporting false positives for functions that were only called from within one of the `deftype` definitions. I found that the functions weren't being used by running `git grep` for each function.